cmd/mdatagen/metrics_v2.tmpl

```diff
@@ -251,7 +251,11 @@ func (mb *MetricsBuilder) Reset(options ...metricBuilderOption) {
 var Attributes = struct {
 {{- range $name, $info := .Attributes }}
 	// {{ $name.Render }} ({{ $info.Description }})
-	{{ $name.Render }} string
+        {{- if $info.Type.ValueType }}
+	{{ $name.Render }} {{ $info.Type.ValueType }}
```

Comment:
According to: https://github.com/open-telemetry/opentelemetry-collector/blob/main/pdata/internal/common.go#L47
This would translate to "BOOL"

I think we need some logic similar to Primitive() in the template? 
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/cmd/mdatagen/loader.go#L85

@dmitryax am I thinking about this correctly?

Reply:
Right, `ValueType` already has `Primitive()` which should be used here